defaults/main.yml

```diff
@@ -191,7 +191,15 @@ nexus_blob_names:           # Splited blob name list @ blob_vars.yml
 nexus_blobstores: []
 # example blobstore item :
 # - name: separate-storage
+#   type: file
 #   path: /mnt/custom/path
+# - name: s3-blobstore
+#   type: S3
+#   config:
+#     bucket: s3-blobstore
+#     accessKeyId: YOUR KEY ID
+#     secretAccessKey: YOUR ACCESS KEY
```

Comment:
I didn't have to to use endpoint and signertype for my config to work. This is my config. 
``` 
nexus_blobstores:
  - name: s3-store
    type: S3
    config:
      bucket: your_bucket
      accessKeyId: your_key_id
      secretAccessKey: your_key
      region: your_regious

```

Reply:
I realise that and I appreciate the contribution, I was just hoping that you'd help to add more documentation as it's otherwise really hard to work out how to configure the plugin. Just a suggestion to make it simpler for users.

Reply:
Hi !
I'm currently crafting the next role release and will embark these changes. Meanwhile, I have to admit I'm quite blind for tests since I do not have an S3 account and I'm not planning to get one as I currently have no use for it. So there is currently no way I can test anything either manually or automagically.

So I'm planning to include this based on your own return, trusting that it worked correctly on your side.

I'm not planning to include any fancy docs regarding support for S3 and will direct people to the official nexus documentation.

Reply:
@zeitounator we have this tested and running in my company's production environment.